cmd/vcluster/cmd/start.go

```diff
@@ -47,10 +46,16 @@ func NewStartCommand() *cobra.Command {
 	}
 
 	options.AddFlags(cmd.Flags(), vClusterOptions)
+	pro.AddProFlags(cmd.Flags(), vClusterOptions)
 	return cmd
 }
 
 func ExecuteStart(ctx context.Context, options *options.VirtualClusterOptions) error {
+	err := pro.ValidateProOptions(options)
+	if err != nil {
```

Comment:
[Reduced scope of variable](https://github.com/uber-go/guide/blob/master/style.md#reduce-scope-of-variables)
```suggestion
	if err := pro.ValidateProOptions(options); err != nil {
```